src/wire/mod.rs

```diff
@@ -85,6 +85,8 @@ pub(crate) mod ip;
 mod ipv4;
 #[cfg(feature = "proto-ipv6")]
 mod ipv6;
+#[cfg(feature = "proto-ipv6")]
+mod ipv6ext;
```

Comment:
nit: for other modules in `wire` if the module is `icmpv4` the exported names are `Icmpv4<something>. Perhaps `ipv6ops` or `ipv6option` would be more in-line with other implementations in `wire`.

Reply:
I chose `ipv6option`. I originally was going to put all of the extension headers in the `ipv6ext` module, but have a module for options seems fine to me.